packages/connector-aliyun-dm/src/single-send-mail.ts

```diff
@@ -0,0 +1,44 @@
+import { Response } from 'got';
+
+import { PublicParameters, request, SendEmailResponse } from './utils';
+
+/**
+ * @doc https://help.aliyun.com/document_detail/29444.html
+ *
+ */
+interface SingleSendMail {
+  AccountName: string;
+  AddressType: '0' | '1';
+  ClickTrace?: '0' | '1';
+  FromAlias?: string;
+  HtmlBody?: string;
+  ReplyToAddress: 'true' | 'false';
+  Subject: string;
+  TagName?: string;
+  TextBody?: string;
+  ToAddress: string;
+}
+
+const Endpoint = 'https://dm.aliyuncs.com/';
+
+const staticConfigs = {
+  Format: 'json',
+  SignatureMethod: 'HMAC-SHA1',
+  SignatureVersion: '1.0',
+  Version: '2015-11-23',
+};
```

Comment:
nit: just curious what static variabled should be placed under the global constants file which not.